src/main/java/redis/clients/jedis/util/JedisBroadcastReplies.java

```diff
@@ -0,0 +1,40 @@
+package redis.clients.jedis.util;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The collection of replies where a command is broadcasted to multiple nodes, but the replies are expected to differ
+ * even in ideal situation.
+ */
+public class JedisBroadcastReplies {
+
+  private static final Object DUMMY_OBJECT = new Object();
+
+  private final Map<Object, Object> replies;
```

Comment:
Why not `Map<HostAndPort, Object> replies;` or NodeID?

Reply:
Because we don't a specified solution for this, e.g. in ConnectionProvider and CommandExecutor interfaces, yet. For example, in ConnectionProvider interface the respective method definition is `Map<?, ?> getConnectionMap()`.